### DISCUSSION TITLE
Limit download speed?

### DISCUSSION THREAD

**Discussion by A-Gift-Of-Flame** [pystardust/ani-cli#1435] [Q&A] [answered]
Hello ani-cli community I am wondering if it is possible to limit download speed when using ani-cli -d, I have already tried to use trickle and it does not appear to function in concert with ani-cli.
I am asking because of a large amount of streaming in my household and my downloads causing stutter across my network due to saturation of download speed.
I will be getting fibre shortly but I'm worried I will saturate that also.

**Comment by CoolnsX** [comment]
nah, fiber are actually good in this. I have fiber connection and it doesn't have problems

**Comment by 71zenith** [comment, marked as answer]
We use aria2c and ffmpeg to download. Both have limit flags. You will need to manually edit the script to add them

**Reply by A-Gift-Of-Flame** [reply]
I will figure out how to do this thanks!
